package.json

```diff
@@ -4,28 +4,29 @@
   "version": "0.0.14",
   "description": "truffle - Simple development framework for Ethereum",
   "dependencies": {
-    "coffee-script": "*",
```

Comment:
John, I'm having trouble noticing what you did here. Did you remove any previous dependencies, or just reorder them? (I see you added some; that's fine)